core/client/example/main.go

```diff
@@ -232,22 +231,16 @@ action get_all() public view {
 // go:embed test.json
 // var testJSON []byte
 
-func unmarshalKf(file io.Reader) (*transactions.Schema, error) {
+func unmarshalKf(file io.Reader) (*types.Schema, error) {
 	source, err := io.ReadAll(file)
 	if err != nil {
 		return nil, fmt.Errorf("failed to read Kuneiform source file: %w", err)
 	}
 
-	astSchema, err := kfparser.Parse(string(source))
+	parseRes, err := parse.ParseKuneiform(string(source))
 	if err != nil {
 		return nil, fmt.Errorf("failed to parse file: %w", err)
 	} // kfSchema := astSchema.(*schema.Schema); j, _ := json.Marshal(kfSchema)
 
-	schemaJSON, err := astSchema.ToJSON()
-	if err != nil {
-		return nil, fmt.Errorf("failed to marshal schema: %w", err)
-	}
-
-	var db transactions.Schema
-	return &db, json.Unmarshal(schemaJSON, &db)
+	return parseRes.Schema, nil
```

Comment:
@brennanjl I hadn't updated the example app when you guys reviewed.  Was forced to do it in this PR in order to update it's deps.  I think this is the intended new way to parse kuneiform.